seqr/views/utils/dataset_utils.py

```diff
@@ -449,3 +450,40 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
         logger.warning(warning, user)
 
     return samples_to_load, info, warnings
+
+
+PHENOTYPE_PRI_HEADER = ['tool', 'project', 'sampleId', 'rank', 'geneId', 'diseaseId', 'diseaseName', 'scoreName1', 'score1']
+MAX_SCORES = 100
+
+
+def _parse_phenotype_pri_row(row):
+    record = {_to_snake_case(key): row[key] for key in PHENOTYPE_PRI_HEADER[:-2]}
```

Comment:
It would be cleaner to define separate lists of validating missing columns and for parsing the row
```
PHENOTYPE_PRI_HEADER = [...]
PHENOTYPE_PRI_REQUIRED_HEADER = PHENOTYPE_PRI_HEADER + ['scoreName1', 'score1']
```